.azure/pipelines/ci-public.yml

```diff
@@ -343,6 +342,7 @@ stages:
                   -sign
                   -buildInstallers
                   -noBuildNative
+                  -publish
```

Comment:
We don't want to pass this for `ci-public`, right?

Reply:
It's totally fine to pass for public CI, and tests the publishing logic in aspnetcore. Most repos do this, and if it works here I think we should use it.